.github/workflows/deploy.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         with:
           inlineScript: |
             echo "Installing containerapp extension"
-            az extension add --source https://workerappscliextension.blob.core.windows.net/azure-cli-extension/containerapp-0.2.0-py2.py3-none-any.whl --yes
+            az extension add --name containerapp --yes

```

Comment:
There's a chance that this might break again if the containerapp commands move to the core CLI. Change this line to `az config set extension.use_dynamic_install=yes_without_prompt` to get the CLI to automatically install extensions if missing. This way if the commands move to core CLI in the future, it just skips the extension installation.